src/core_plugins/kibana/public/context/query_parameters/__tests__/action_set_query_parameters.js

```diff
@@ -0,0 +1,50 @@
+import expect from 'expect.js';
+
+import { createStateStub } from './_utils';
+import { QueryParameterActionsProvider } from '../actions';
+
+
+describe('context app', function () {
+  describe('action setQueryParameters', function () {
+    it('should update the queryParameters with certain values from the given object', function () {
+      const { setQueryParameters } = new QueryParameterActionsProvider();
+      const state = createStateStub({
+        queryParameters: {
+          additionalParameter: 'ADDITIONAL_PARAMETER',
+        }
+      });
+
+      setQueryParameters(state)({
+        anchorUid: 'ANCHOR_UID',
+        columns: ['column'],
+        defaultStepSize: 3,
+        indexPattern: 'INDEX_PATTERN',
+        predecessorCount: 100,
+        successorCount: 100,
+        sort: ['field'],
+      });
+
+      expect(state.queryParameters).to.eql({
+        additionalParameter: 'ADDITIONAL_PARAMETER',
+        anchorUid: 'ANCHOR_UID',
+        columns: ['column'],
+        defaultStepSize: 3,
+        indexPattern: 'INDEX_PATTERN',
+        predecessorCount: 100,
+        successorCount: 100,
+        sort: ['field'],
+      });
+    });
+
+    it('should ignore additional new values', function () {
```

Comment:
This description isn't very clear to me. How does "should ignore invalid properties" sound to you?